docs/reference/migration/migrate_8_0/scripting.asciidoc

```diff
@@ -22,4 +22,29 @@ scripts. Any use of `getDayOfWeek` expecting a return value of `int` will result
 in a compilation error or runtime error and may not allow the upgraded node to
 start.
 ====
+
+.Empty scripts/templates are no longer allowed as stored scripts.
+[%collapsible]
+====
+*Details* +
+Empty scripts/templates cannot be specified as the source of a
+PutStoredScript action.
+
+*Impact* +
+Before upgrading, remove stored scripts/templates that are empty. Otherwise,
+these scripts will be dropped, and may not align with the scripts on older
+nodes.
```

Comment:
I added a link to the delete stored script API. I assume the user will use that API to remove the scripts.

```suggestion
Before upgrading, use the {ref}/delete-stored-script-api.html[delete stored
script API] to delete any empty stored scripts or search templates.

In 8.0, {es} will drop any empty stored scripts or empty search templates from
the cluster state. Requests to create a stored script or search template with
an empty `source` will return an error.
```

> and may not align with the scripts on older nodes.

My suggestion doesn't include this because I wasn't sure what it meant. Does it just mean that behavior will be different after the upgrade?

Reply:
Yes. Poor wording on my part. I'm adopting your suggestion as the entirety of the text here.